modules/dag-shared/src/main/scala/org/tessellation/dag/snapshot/StateChannelSnapshotBinary.scala

```diff
@@ -8,8 +8,9 @@ import org.tessellation.security.hash.Hash
 
 import derevo.circe.magnolia.{decoder, encoder}
 import derevo.derive
+import derevo.scalacheck.arbitrary
 
-@derive(encoder, decoder)
+@derive(arbitrary, encoder, decoder)
```

Comment:
can it be an arbitrary if it must keep the chain?

Reply:
Removed.

Reply:
that was just a question :P